### DISCUSSION TITLE
"Theme" in the title breaks revealjs presentation

### DISCUSSION THREAD

**Discussion by royfrancis** [quarto-dev/quarto-cli#5455] [Q&A] [answered]
The slide jumps to first slide when landing on the slide with heading "Theme". See [demo](https://www.royfrancis.com/quarto-exp/documents/theme-title/).

Example:

```
---
title: Testing theme title
date: 08-May-2023
format:
  revealjs
---

## Slide 1

Content

## Theme

Some more content

## Bla

More stuff here
```

`quarto render index.qmd`

```
🢒  quarto --version
1.3.340
```

**Comment by mcanouil** [comment]
Edit: The issue is not with the title, but with the format key which is somehow not parsed very well.

~~The issue is not with the title but with your `revealjs` key that has no value, somehow it is still parsed but not well~~.

Correct/Working yaml are:
```yaml
format: revealjs
```

```yaml
format:
  revealjs: default
```

**Reply by mcanouil** [reply]
There might be something, we are looking more into this.

**Reply by mcanouil** [reply]
@royfrancis Are you using VSCode?

**Comment by mcanouil** [comment]
@royfrancis It's indeed a bug in Quarto, currently being worked on. A fix should be available soon.
Thanks for the report.

**Reply by mcanouil** [reply, marked as answer]
Fixed in 5d2f800b7ac0a5f4bf787bd3c60d59e9367147eb, should be available in the next pre-release build and later in a patch to 1.3.
